_gabarits/base-m88.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <!-- Adapter le titre -->
+    <title>Nom Page - Nom section - Nom projet - Mandat</title>
+    <!-- Feuille de styles commune à tous -->
+    <link rel="stylesheet" href="../css/styles.css">
+    <!-- Ajuster le href pour pointer sur la feuille de styles
+    du mandat en changeant le suffixe m99 -->
```

Comment:
Il faudrait changer toutes les références au mandat 99 pour ton mandat 88.
Tu peux faire un Find and Replace dans PHPStorm.